worker/worker.go

```
@@ -0,0 +1,54 @@
+package worker
+
+import (
+	"encoding/json"
+	"github.com/ufcg-lsd/arrebol-pb-worker/utils"
+	"log"
+	"os"
+)
+
+type PBWorker struct {
+	ServerEndPoint string
+	Vcpu           string
+	Ram            string
+	Image          string
+	Address        string
+	Token          string
+	Id             string
+	QueueId        string
+}
+
+func (w *PBWorker) Subscribe() {
+	httpResponse := utils.Post(w.Id, &PBWorker{Ram: w.Ram, Vcpu: w.Vcpu, Image: w.Image,
+		Address: w.Address, Id: w.Id, QueueId: w.QueueId}, w.ServerEndPoint + "/workers")
+
+	if httpResponse.StatusCode != 201 {
+		log.Fatal("The work could not be subscribed")
+	}
+
+	var parsedBody map[string]string
+	json.Unmarshal(httpResponse.Body, &parsedBody)
```

Any reason to ignore the error?